### PR TITLE
Add configurable extra containers

### PR DESCRIPTION
**Pull request by akowal**

### CODE REVIEW

**Review by nickjn92 — APPROVED** [review]

File: charts/app/values.yaml [hunk 1]
```diff
@@ -66,3 +66,17 @@ global:
     enabled: true
     port: 8081
     path: /actuator/prometheus
+
+  cloudSqlProxy:
+    enabled: false
+    instances: <project>:<locaiton>:<db>=tcp:<port>
```
Comment (nickjn92) [inline charts/app/values.yaml:72]: typo in `location`

Comment (akowal) [reply charts/app/values.yaml:72]: Fiexd =)

Comment (nickjn92) [reply charts/app/values.yaml:72]: Also is it better to allow free form string or build the string using project, location, db, port etc as defined keys?
eg
```yaml
project: pej-dev
location: ...
db: ...
port: 5432
```

Comment (akowal) [reply charts/app/values.yaml:72]: I thought about it, but decided it's not worth it, two reasons:
1) final string is basically a concatenation of all these parameters, saving a few `:` chars is not worth it, imo
2) I suspect it's called `instances` (plural) for a reason and there might be some use for that later on

Comment (nickjn92) [reply charts/app/values.yaml:72]: Alright, lets keep as. Seems the typo on location fix hasnt been pushed?

Comment (akowal) [reply charts/app/values.yaml:72]: My bad, pushed.

**Review by nickjn92 — APPROVED** [review]

**Pull request merged**